src/agor/strategy.py

```
@@ -116,14 +116,13 @@ def sync_state(self) -> None:
 
         try:
             # Pull latest changes
-            result = subprocess.run(
-                ["git", "pull"], capture_output=True, text=True, cwd=self.project_root
-            )
+            success, output = run_git_command(["pull"], cwd=str(self.project_root))
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'cwd' in function call

(E1123)